src/interfaces/IPoolManager.sol

```diff
@@ -120,6 +120,9 @@ interface IPoolManager is IFees, IERC6909Claims {
     /// @notice Returns the reserves for a given ERC20 currency
     function reservesOf(Currency currency) external view returns (uint256);
 
+    /// @notice Returns whether the contract is locked
+    function isLockSet() external view returns (bool);
```

Comment:
mm my instinct is that i prefer `isLocked` for this. Although we'll be renaming it in a later PR anyway so it doesnt really matter 😆

Reply:
this function will be removed when we have extsload also just a note

Reply:
so its probably nbd rn and since we have isLocked already I think its a fine temp name

Reply:
yeah good point lets just leave it as is